releasenotes/notes/SNMP---include-traps-db-7c44bd129daf7667.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    Include pre-generated trap db file in the `conf.d/snmp.d/traps_db/` folder.
```

```suggestion
    Include pre-generated trap db file in the ``conf.d/snmp.d/traps_db/`` folder.
```

Since the fragment will be used in .rst file, you need double tick.

> backquotes: \`\`text\`\` for code samples.